composer.json

```diff
@@ -9,15 +9,15 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": "^8.1",
-        "silverstripe/framework": "^5",
+        "silverstripe/framework": "^6",
         "monolog/monolog": "^3.2",
         "psr/log": "^3",
         "silverstripe/silverstripe-proxy-db": "^2"
```

Comment:
proxy-db will need a bump too for composer to be happy (or we need to ditch it outright)

Reply:
Updated